src/context.rs

```diff
@@ -559,11 +559,12 @@ pub enum Shell {
 }
 
 /// Which kind of prompt target to print (main prompt, rprompt, ...)
-#[derive(Debug, Clone, Copy, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
 pub enum Target {
     Main,
     Right,
     Continuation,
+    Profile(String),
```

Comment:
`String` isn't `Copy`, so it makes sense that `Copy` is no longer derivable on `Target`.

Removing `Copy` from things can sometimes result in huge ergonomic pains, but based on the fact that your code compiles and I haven't seen a `.clone()` explosion yet, I'm going to guess that it wasn't *that* bad in this case.

Reply:
That was my conclusion as well. I was afraid things would start to badly fail.